lib/cocoapods/generate/installer.rb

```diff
@@ -275,9 +284,9 @@ def add_test_spec_schemes_to_app_scheme(installer, app_project)
         end
       end
 
-      def make_ios_app_launchable(installer, app_project, native_app_target)
+      def make_ios_app_launchable(app_project, native_app_target)
         platform_name = Platform.string_name(native_app_target.platform_name)
-        generated_source_dir = installer.sandbox.root.join('App', platform_name).tap(&:mkpath)
+        generated_source_dir = install_directory.join("App-#{platform_name}").tap(&:mkpath)
```

Comment:
fixed an issue in which the info.plist and launch storyboard we add is in the wrong folder.